documentation/sphinx/source/consistency-check-urgent.rst

```diff
@@ -0,0 +1,87 @@
+##############################
+Consistency Checker Urgent
+##############################
+
+| Author: Zhe Wang
+| Reviewer: Jingyu Zhou
+| Audience: FDB developers, SREs and expert users.
+
+In a FoundationDB (FDB) key-value cluster, every key-value pair is copied across multiple storage servers. 
+The Consistency Checker Urgent tool is used to validate the consistency of all replica for each key-value pair. 
+If any data inconsistency is detected, the tool generates ConsistencyCheck_DataInconsistent trace events for the corresponding shard. 
+There are two types of data inconsistencies: 
+
+1. Value mismatch, where the value of a key on one server differs from that on another server, 
+2. Unique key, where a key exists on one server but not on another. 
+
+The ConsistencyCheck_DataInconsistent trace event helps differentiate between these two types of corruption.
+
+Key features
+============
+The Consistency Checker Urgent tool is designed to ensure safe, fast, and comprehensive checking of data consistency across the entire key space 
+(i.e., " " ~ "\xff\xff"). It achieves this through the following features:
```

Comment:
is this true? `\xff` key space is also checked (I guess the default is not including it)?

Reply:
Yes. It is true. See: CONSISTENCY_CHECK_URGENT_RANGE_END_0